src/test/java/org/jabref/gui/fieldeditors/LinkedFileViewModelTest.java

```diff
@@ -23,8 +23,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
please leave the imports as they were before (we have the convention not to use the `*` import matcher). You can also import our code style into IntelliJ: https://github.com/JabRef/jabref/tree/master/config